src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -159,6 +160,10 @@ export const ProjectOverview: FC = () => {
     },
   });
 
+  const { data: financialReportsData } = useQuery(FinancialReportsDocument, {
+    variables: { projectId, changeset: changesetId },
+  });
```

Comment:
Let's just add the data we need to the existing ProjectOverview query instead of adding another one.